.rspec

```diff
@@ -0,0 +1,3 @@
+--require spec_helper
```

Comment:
looks like we are using rspec instead of rack::test ?

Reply:
Good catch looks like we didn't end up needing that!